instrumentation/otel-webserver-module/src/apache/ApacheHooks.cpp

```diff
@@ -49,6 +51,8 @@ const char* APPD_OUTPUT_FILTER_NAME = "APPD_EUM_AUTOINJECT";
 
 appd::core::WSAgent wsAgent; // global variable for interface between Hooks and Core Logic
 
+using namespace appd::core::sdkwrapper;
```

Comment:
Why did we need to call this?

Reply:
Using namespace is used to use the variables in that namespace directly